zim.py

```diff
@@ -53,7 +53,10 @@
 try:
 	encoding = sys.getfilesystemencoding() # not 100% sure this is correct
 	argv = [arg.decode(encoding) for arg in sys.argv]
-	exitcode = zim.main.main(*argv)
+	if __name__ == '__main__':
```

Comment:
What is the purpose of the `__name__ ==  '__main__'` wrapper here ?

This file is always run as the main program.